app/controllers/admin/merchants_controller.rb

```diff
@@ -8,13 +8,27 @@ def show
     @merchant = Merchant.find(params[:id])
   end
 
+  def new
+  end
+
+  def create
```

Comment:
Think this was a good idea to move the admin merchant controller, ensures we have our admin routes having a single responsibility